packages/kbn-es/src/settings.ts

```diff
@@ -0,0 +1,63 @@
+/*
```

Comment:
__note:__ I see we call these settings as `esArgs`, but I picked the word `settings` as it's used in [`Secure Settings`](https://www.elastic.co/guide/en/elasticsearch/reference/current/secure-settings.html).